.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+*                     @mozilla-services/context-services
```

Comment:
Hi @ncloudioj @pjenvey @jrconlin! 👋🏻 

Any objections to requesting review from the `@mozilla-services/context-services` team by default? Currently we manually request review. Defining GitHub Code Owners will do this automatically.

Reply:
Not at all, this looks reasonable to me 👍